drafts/draft-0002.md

```diff
@@ -0,0 +1,131 @@
+## Preamble
+
+```
+SEP: Not assigned
+Title: Anchor/Client interoperability
+Author: stellar.org
+Status: Draft
+Created: 2017-10-30
+```
+
+## Simple Summary
+It will be easier for users if wallets and other clients can interact with anchors directly without the user having to leave the wallet to go to the anchor's site. This SEP is intended to create a standard way that anchors allow this to be done.
+
+## Abstract
+Proposal for a standard protocol that allows wallets to deposit and withdraw funds info from anchors.
+
+
+## Specification
+
+
+### Deposit
+
+This is the user getting a token from the anchor. Deposits will be a new endpoint on the federation server. It will just return the accountId where we should send the currency that we specified. We will get the equivalent Asset issued by the anchor in return.
+
+`DEPOSIT_SERVER` should be specified in the stellar.toml of the anchor.
+
+Endpoint: `DEPOSIT_SERVER/deposit`<br>
+Purpose: Get intructions for depositing an asset with the anchor.<br>
+Method: GET<br>
```

Comment:
`/deposit` should be a `POST` endpoint since anchor will need to prepare itself to receive a deposit by creating an entry in it’s db with the address it returns (binding it to the stellar address sent in the request).